packages/resilience/README.md

```diff
@@ -21,8 +21,7 @@ Add a timeout middleware to a request.
 ```
 executeHttpRequest(destination, { middleware: [ timeout() ] }, httpRequestOptions);
 ```
-<!-- TODO v3 put better link when documentation is done -->
-For more detailed overview visit our [resilience documentation](https://sap.github.io/cloud-sdk/docs/js/).
+For more detailed overview visit our [resilience documentation](https://sap.github.io/cloud-sdk/docs/js/v3/guides/resilience).
```

Comment:
[q] Shouldn't the path be `/docs/js/guides/resilience`, since we'll eventually make v3 docs default, and the same path will work (without /v3/).

Reply:
yes you are right. In the migration guide the link should be v3 to ensure that you get to the v3 docs. But here it should be the latest one.